tests/lib/rules/no-children-prop.js

```diff
@@ -15,6 +15,7 @@ var RuleTester = require('eslint').RuleTester;
 var parserOptions = {
   ecmaVersion: 6,
   ecmaFeatures: {
+    experimentalObjectRestSpread: true,
```

Comment:
I'm not 100% on it either way - it just seems like it'd be useful to test the parser with common (non-experimental) options.